app/slayherobrine/MinecrafterDeclaration.jsx

```diff
@@ -54,23 +54,22 @@ const MinecrafterDeclaration = () => {
           <h2 className="book-title">THE MINECRAFTER DECLARATION OF INDEPENDENCE</h2>
```

Comment:
```suggestion
          <h2 className="book-title">THE MINECRAFTER DECLARATION&nbsp;OF&nbsp;INDEPENDENCE</h2>
```
this will make it look more like
> THE MINECRAFTER
> DECLARATION OF INDEPENDENCE

Reply:
also would recommend
> The Minecrafter Declaration of Independence

and use CSS to uppercase